tasks.yaml

```diff
@@ -30,6 +30,9 @@ tasks:
       - description: "Deploy the Istio source package with Zarf Dev"
         cmd: "uds zarf dev deploy src/istio --flavor ${FLAVOR}"
 
+      - description: "Deploy the Prometheus-Stack source package with Zarf Dev to only install the CRDs"
+        cmd: "uds zarf dev deploy src/prometheus-stack --flavor crds-only"
+
```

Comment:
Nope, `dev-setup` is broken on main - Pepr will fail for anything needing a service monitor (like `keycloak`) - this abuses flavors to just put the crds in the cluster so Pepr can continue with it's mutations.

Reply:
Ah, okay tracking. I missed the first time around that this was an addition to the dev-setup task. Honestly might be good to revisit in general and see what makes sense for dev-setup...maybe good to look at this the same way we do slim-dev (i.e. adding keycloak to it as well). Probably a separate discussion there...

I'm hesitant to add a `--flavor` that doesn't exist, at least without adding a clarifying/explanatory comment on what is happening here. This would behave the same if we just dropped `--flavor` entirely, correct? I don't mind using/abusing flags like this but I don't think it's super apparent why it's there at first glance.

Reply:
Correct - I added the flavor flag to more indicate that it is an abuse but will clarify that in a comment as well